source/api/search/2.0/index.md

```diff
@@ -0,0 +1,777 @@
+
+The annotations may also include references to the structure or structures that the target (the resource in the `target` property) is found within.  The URI and type of the including resource _MUST_ be given, and a `label` _SHOULD_ be included.
+
+This structure is called out explicitly as although it uses only properties from the Presentation API, it is not a common pattern and thus clients may not be expecting it.
+
+``` json-doc
+{
+  "@context":"http://iiif.io/api/search/{{ page.major }}/context.json",
+  "id":"http://example.org/service/manifest/search?q=bird&motivation=painting",
+  "type":"AnnotationPage",
+
+  "items": [
+    {
+      "id": "http://example.org/identifier/annotation/anno-line",
+      "type": "Annotation",
+      "motivation": "painting",
+      "body": {
+        "type": "TextualBody",
+        "value": "A bird in the hand is worth two in the bush",
+        "format": "text/plain"
+      },
+      "target": {
+        "id": "http://example.org/identifier/canvas1#xywh=100,100,250,20",
+        "partOf": {
+          "id": "http://example.org/identifier/manifest",
+          "type": "Manifest",
+          "label": "Example Manifest"
+        }
+      }
+    }
+    // Further annotations here ...
+  ]
+}
+```
+
+### 3.4 Search API Specific Responses
+{: #search-api-specific-responses}
+
+There may be properties that are specific to the search result, and not features of the annotation in general, that are valuable to return to the client.  Examples of such properties include the text before and after the matched content (to allow a result snippet to be presented), the matched text itself (when case normalization, stemming or wildcards have been applied), and a reference to the set of annotations that together fulfill the search query (when a phrase spans across multiple annotations).
+
+As these responses include Search specific information, the value of `@context` _MUST_ be an array with both the Presentation API and the Search API context URIs included, in that order.  This allows the two APIs to develop separately and yet remain as synchronized as possible.
+
+To incrementally build upon existing solutions and provide graceful degradation for clients that do not support these features and retain compatibility with the Presentation API, the search API specific information is included in a second list within the annotation list called `hits`, other than the `ignored` property on the layer.  Annotation lists _MAY_ have this property, and servers _MAY_ support these features.
+
+If supported, each entry in the `hits` list is a `search:Hit` object.  This type must be included as the value of the `type` property. Hit objects reference one or more annotations that they provide additional information for, in a list as the value of the hit's `items` property.  The reference is made to the value of the `id` property of the annotation, and thus annotations _MUST_ have a URI to enable this further information.
```

Comment:
We talked about changing the "must" in this sentence to normative: This type `must` be included as the value of the `type` property.

Reply:
I've pushed an update to fix this, and also remove the reference to "layer" in the first paragraph, since this concept is deprecated as of Prezi3.